playbooks/roles/rpc_pre_upgrade/tasks/main.yml

```diff
@@ -16,6 +16,30 @@
 # set backup_dir as a fact to ensure dir name doesn't change during run
 - set_fact:
     backup_dir: "{{ ansible_env.BKUPDIR |default(local_home+'/'+'rpc-upgrade-'+date_stamp) }}"
+    marker_dir: "{{ ansible_env.UPGRADE_LEAP_MARKER_FOLDER | default('/etc/openstack_deploy/upgrade-leap) }}"
```

Comment:
We might need to create the /etc/openstack_deploy/upgrade-leap directory since it's likely it doesn't exist yet.

Reply:
OSA ops did create the directory but I'm not sure the timing is soon enough for this role
https://github.com/openstack/openstack-ansible-ops/blob/052bdbf672dc72a9513c9103f80f76fcb7503ada/leap-upgrades/lib/functions.sh#L76

Reply:
Miss right single quote around default path.
`default('/etc/openstack_deploy/upgrade-leap')`